cms-contentful/src/main/java/com/commercetools/sunrise/cms/contentful/models/FieldType.java

```diff
@@ -0,0 +1,74 @@
+package com.commercetools.sunrise.cms.contentful.models;
+
+import com.contentful.java.cda.CDAAsset;
+import com.contentful.java.cda.CDAField;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public enum FieldType {
+    BOOLEAN("Boolean", true),
+    DATE("Date", true),
+    INTEGER("Integer", true),
+    NUMBER("Number", true),
+    SYMBOL("Symbol", true),
+    TEXT("Text", true),
+    LOCATION("Location", true),
+    LINK("Link", false),
+    ASSET("Asset", false),
+    ARRAY("Array", false);
+
+    private static final Set<String> WITH_STRING_REPRESENTATION = Collections.unmodifiableSet(
+            Arrays.stream(values())
+                    .filter(contentType -> contentType.hasStringRepresentation)
+                    .map(FieldType::type)
+                    .collect(Collectors.toSet()));
+
+    private final String type;
+    private final boolean hasStringRepresentation;
+
+    FieldType(String type, boolean hasStringRepresentation) {
+        this.type = type;
+        this.hasStringRepresentation = hasStringRepresentation;
+    }
+
+    public String type() {
+        return type;
+    }
+
+    public static Function<Object, String> getToStringStrategy(final CDAField contentType) {
```

Comment:
That `get` makes it a bit harder to read IMO, just `toStringStrategy` sounds good enough.